### DISCUSSION TITLE
Deleting 2 or more items fails

### DISCUSSION THREAD

**Discussion by AlexVanK** [directus/directus#13520] [Archive]
Hi,
not sure if this is a bug or a problem in my setup...

But I can't delete 2 or more items at the same time....
![image](https://user-images.githubusercontent.com/56647386/170107459-8349a326-10f4-428d-957f-ab562817aece.png)

Note that **document_file_id** is a relational field... this is the setup:
![image](https://user-images.githubusercontent.com/56647386/170107709-25dbe69b-57be-4048-be2b-1c88099b3f15.png)

**Comment by rijkvanzanten** [comment]
Very hard to tell based on the limited information, though based on the error message show, I think it has something to do with the relational cascade of a many to many field. Could you share some more information on your schema setup so we can try to reproduce this in isolation?

**Reply by AlexVanK** [reply]
Hi @rijkvanzanten 

This is my current collection with two items in it:
![image](https://user-images.githubusercontent.com/56647386/170118229-1dc88411-f169-4f70-a495-5db23a677fb4.png)

document_file_id is a "file" relational field, set up as follows:
![image](https://user-images.githubusercontent.com/56647386/170118968-6f42f55a-d386-4b06-931b-20ea133adea7.png)
![image](https://user-images.githubusercontent.com/56647386/170118432-2c97f338-2698-4538-b310-1a6b4a9d8975.png)

The deletion fails whether I do it manually from the admin interface, or from the SDK...
Not sure how to provide you with more information about the schema, let me know.

**Reply by rijkvanzanten** [reply]
@azrikahar could you try this out locally?

**Reply by azrikahar** [reply]
> @azrikahar could you try this out locally?

@rijkvanzanten Gave it a try and I can't seem to reproduce it:

https://user-images.githubusercontent.com/42867097/170658142-6e00fb8c-4884-4c3d-a325-d3b8e3e355ae.mp4

> Not sure how to provide you with more information about the schema, let me know.

@AlexVanK  A schema snapshot would be helpful! Along with the info usually requested in issues, such as your Directus version, database used, how you are currently deploying/running Directus etc.

**Reply by AlexVanK** [reply]
Guys, I figured out the problem... 
I had a custom HOOK polluting the input variable... hence all was falling apart.

Thank you for your help anyway!
